config/services.yaml

```diff
@@ -59,8 +59,16 @@ parameters:
 
     html5_validation: true
 
-    front_sort_options: top|hot|active|newest|oldest|commented
-    front_time_options: 6h|12h|1d|1w|1m|1y|all|∞
+    front_sort_options: top|hot|active|newest|oldest|commented # TODO remove fallback after tag rework
```

Comment:
decided to keep this for now as I don't want to touch the tag route file while the tag rework is in progress, we can change it to `default_sort_options` (and optionally add any other filters) later